thingsboard_gateway/extensions/tcpserver/tcpserver_connector.py

```diff
@@ -0,0 +1,243 @@
+#     Copyright 2020. ThingsBoard
+#
+#     Licensed under the Apache License, Version 2.0 (the "License");
+#     you may not use this file except in compliance with the License.
+#     You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#     Unless required by applicable law or agreed to in writing, software
+#     distributed under the License is distributed on an "AS IS" BASIS,
+#     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#     See the License for the specific language governing permissions and
+#     limitations under the License.
+
+
+import threading
```

Comment:
Please use the particularly import instead import of the all module (`from threading import Thread` instead of `import threading`). It will optimize loading.

Reply:
ok